First version on how to bring your custom kube-prometheus-stack (#206)
kyma-project/examples: monitoring-custom-metrics/README.md

    export KYMA_EXAMPLE_NS="{namespace}"
    ```

> 2. Ensure that your namespace is having Istio sidecar injection enabled, the example will assume that the metrics are exposed ina strict mTLS mode:

Reviewer: ```suggestion
2. Ensure that your Namespace has Istio sidecar injection enabled. This example assumes that the metrics are exposed in a strict mTLS mode:

```

    export KYMA_EXAMPLE_NS="{namespace}"
    ```

2. Ensure that your Namespace has Istio sidecar injection enabled. This example assumes that the metrics are exposed in a strict mTLS mode:

   ````
   kubectl label namespace ${KYMA_EXAMPLE_NS} istio-injection=enabled

3. Deploy the service:

    ```bash
    kubectl apply -f deployment/deployment.yaml -n $KYMA_EXAMPLE_NS
    ```

4. Deploy the ServiceMonitor:

    ```bash

>     kubectl apply -f deployment/service-monitor.yaml

Reviewer: In line 53 (can't add a comment there), replace `scrapped` with `scraped`.

[to scrap](https://www.merriam-webster.com/dictionary/scrap), scrapped: abandon, discard
[to scrape](https://www.merriam-webster.com/dictionary/scrape), scraped: to collect

Author: done
